src/main/scala/org/scalameter/Persistor.scala

```diff
@@ -6,15 +6,15 @@ package org.scalameter
 
 
 trait Persistor {
-  def load(context: Context): History
-  def save(context: Context, h: History): Unit
+  def load[T: Manifest](context: Context): History[T]
+  def save[T](context: Context, h: History[T]): Unit
```

Comment:
Well, we have already `pickler` embeded in the `Measurement` so I suppose that we could simply read it using `Jackson` as a `History[_]]` and than cast it to the desired type, but I will check that if any problem could occur.